packages/stark-ui/src/modules/slider/components/slider.component.ts

```diff
@@ -54,21 +54,21 @@ export class StarkSliderComponent extends AbstractStarkUiComponent implements Af
 	 * Configuration object for the slider instance to be created.
 	 */
 	@Input()
-	public sliderConfig: StarkSliderConfig;
+	public sliderConfig: StarkSliderConfig = { range: { max: [], min: [] } };
 
 	/**
 	 * HTML "id" attribute of the element.
 	 */
 	@Input()
-	public sliderId: string;
+	public sliderId = "undefined";
 
 	/**
 	 * Array of numeric values to be set to the slider.
 	 * For simple sliders with just one handle, the array should contain only one value.
 	 * For range sliders, the array should contain two or more values.
 	 */
 	@Input()
-	public values: number[];
+	public values: number[] = [];
```

Comment:
I'm not sure if this change will cause a regression because it is compared to the `latestUnencodedValues` in the `ngOnChanges`...

Reply:
What do you suggest I use here? I'm not really familiar with this component and what might be expected behaviour. 😕

Reply:
I'll have a look on this and let you know if this should be changed 😉

Reply:
I changed this to a definite with a onInit check in the meantime. 😄